test/e2e/node/runtimeclass.go

```diff
@@ -58,8 +58,9 @@ var _ = SIGDescribe("RuntimeClass", func() {
 			labelFooName: "bar",
 		}
 		_, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(context.TODO(), pod, metav1.CreateOptions{})
-		framework.ExpectError(err, "should be forbidden")
-		framework.ExpectEqual(apierrors.IsForbidden(err), true, "should be forbidden error")
+		if !apierrors.IsForbidden(err) {
```

Comment:
will `apierrors.IsForbidden(err)` return `false` for `nil`?

Reply:
Yes. It uses errors.As under the hood. None of the API comments talk about nil directly, but the semantic is covered (nil is not assignable to a non-nil status type, so errors.As returns false) and the implementation checks for it.

Reply:
Other code is playing it safe and explicitly checks for nil beforehand, but that's unnecessary.

Reply:
I'm tempted to submit a patch both to k/k and the standard "errors" package with an explicit comment clarification...

Reply:
https://github.com/kubernetes/kubernetes/pull/106646